### DISCUSSION TITLE
Deploy dev and prod docs to github pages

### DISCUSSION THREAD

**Discussion by SteveRosam** [mkdocs/mkdocs#3030] [General]
Anyone know how to deploy both dev and prod docs to github pages?
I have tried various experiments, but the gh-deploy seems to take over and i only get docs on the github-pages environment.

**Comment by Andre601** [comment]
You can just `mkdocs build` the docs and manually push them.

**Comment by fu-sen** [comment]
Do you want to publish the pre-build source with `git push`?
Now that you can build GitHub Pages via GitHub Actions, you might be able to use it. I put a tip for that here:

https://github.com/mkdocs/mkdocs/wiki/Mkdocs-Deployments#github-pages
